requirements.txt

```
@@ -1,7 +1,7 @@
 requests>=2.20.0
 keyring>=8.0,<=9.3.1
 keyrings.alt>=1.0,<=3.2.0
-click>=6.0,<=7.1.1
+click>=8.0
```

No need to up the lower bound if that still works. Let's just remove the upperbound.
```suggestion
click>=6.0
```